src/planscape/forsys/forsys_request_params.py

```diff
@@ -331,6 +339,10 @@ class ForsysGenerationRequestParamsFromUrlWithDefaults(
     _DEFAULT_PRIORITIES = ['fire_dynamics',
                            'forest_resilience', 'species_diversity']
 
+    # Constants for constraining projects
+    _URL_MAX_AREA_PER_PROJECT = 'max_area_per_project'
```

Comment:
nit: can these be moved to line 335 so that it's beneath the section, `# Constants for parsing url parameters`?

Reply:
Done.